bot.js

```diff
@@ -9,7 +9,7 @@ if (process.env.TEST) {
   console.log(`[${process.env.USER} => ${config.name}]`);
 } else {
   const IrcClient = require('./interfaces/irc');
-  var client = new IrcClient(config.server, config.name, config.channels);
+  client = new IrcClient(config.server, config.name, config.channels);
```

Comment:
Yes, I used `var` precisely because of the same reason. Maybe we should use `let` to declare it outside and assign inside the if loops?